resource/style.css

```diff
@@ -0,0 +1,692 @@
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    box-sizing: border-box;
+}
+
+
+
+.header-section{
+    width: 100%;
+    display: flex;
+    padding: 5px 10px 2px 20px;
+    border-bottom: 1px solid rgb(226, 226, 226);
+    position: fixed;
+    background-color: white;
+    top: 0px;
+
+}
+
+.header-section img{
+    max-width: 100%;
+}
+
+a:link {
+    text-decoration: none;
+    cursor: pointer;
+  }
+
+button:hover{
+    cursor: pointer;
+}
+
+#menu{
+    padding: 10px 20px 0 0;
+    flex-wrap: wrap;
+}
+
+#search{
+    padding: 10px 20px 0 10px;
+    flex-wrap: wrap;
+}
+
+#type{
+    padding: 10px 20px 0 10px;
+    font-size: 18px;
+    font-weight: bold;
+    color: black;
+    flex-wrap: wrap;
+}
+
+#newyork{
+  margin: 0 auto;
+  flex-wrap: wrap;
+
+}
+
+#subscribe{
+    padding: 8px 20px 8px 20px;
+    color: white;
+    background: rgb(98, 136, 165);
+    border-radius: 6px;
+    font-size: 14px;
+    font-weight: bold;
+    display: inline;
+    flex-wrap: wrap;
+}
+
+#login{
+    padding: 8px 20px 8px 20px;
+    color: white;
+    background: rgb(98, 136, 165);
+    border-radius: 6px;
+    font-size: 14px;
+    font-weight: bold;
+    display: inline;
+    flex-wrap: wrap;
+}
+
+#button{
+    padding: 0 20px 0 10px ;
+    flex-wrap: wrap;
+}
+
+/* @media (min-width: 1200px){
```

Comment:
Remove any styling you're not using. Don't leave unused stylings in your code as a comment anymore